openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -50,7 +50,7 @@ The audience of the document is implementers that require a high level of securi
 
 # Terminology
 
-This specification uses the terms "Holder", "Issuer", "Verifier", "Wallet", and "Verifiable Credential" as defined in @!OIDF.OID4VCI] and [@!OIDF.OID4VP].
+This specification uses the terms "Holder", "Issuer", "Verifier", "Wallet", "Wallet Attestation", "Credential Type" and "Verifiable Credential" as defined in @!OIDF.OID4VCI] and [@!OIDF.OID4VP].
```

Comment:
I think we should split this into terminology defined by specific specs ( termy x,y,z in VCI and a,b,c in VP) similar to how it is done in VCI and VP currently.

Reply:
@c2bo: Not sure what you are asking for here. I did a side-by-side comparison of the terms of the Terminology sections of the VP and VCI specs: there is a lot of overlap, but not 100%. Perhaps you are saying that a reference to clause 2 is needed in the references in this text? In any case, I think the first part of this this sentence should read "This specification uses **terms such as**..."